src/sections/dataset/dataset-files/useGetFilesCountInfo.ts

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { getFilesCountInfoByDatasetPersistentId } from '../../../files/domain/useCases/getFilesCountInfoByDatasetPersistentId'
+import { FilesCountInfo } from '../../../files/domain/models/FilesCountInfo'
+import { FileCriteria } from '../../../files/domain/models/FileCriteria'
+import { FileRepository } from '../../../files/domain/repositories/FileRepository'
+import { DatasetVersion } from '../../../dataset/domain/models/Dataset'
+
+type UseLoadFilesCountInfoParams = {
```

Comment:
I've thought about it too, and maybe even the word "loaded", "shown" or "displayed" might be more appropriate, since it's possible that you load the next 10 files but don't see them yet.
So something like "10 of 20 files/datasets loaded" or "10 of 20 files/datasets shown" or "10 of 20 files/datasets displayed"
What do you think?

Reply:
Hi @GermanSaracca, I think those options, (loaded, shown, displayed) are equally good, I can't decide which I like better! 😊.  And I couldn't find a definitive UX recommendation. Just to pick one, how about 'displayed'?

Reply:
Great, thanks!  For the merge conflicts, should that be resolved now, or in the QA phase?

Reply:
Merge conflicts resolved and requested changes done :)